cli/tests/tls/README

```diff
@@ -0,0 +1,4 @@
+Certificates generated using instructions from this gist:
+https://gist.github.com/cecilemuller/9492b848eb8fe46d462abeb26656c4f8
+
+They are valid for 3 years.
```

Comment:
Can you just copy and paste those instructions into this readme, so that link doesn't go 404. 

Also please state the absolute expiration date at the top of this readme (EG "Expires: January 1 2022")

It would be very helpful if you could update these certs to expire 99 years in the future instead of 3.

Reply:
```
This certificates will expire on Sept, 27th, 2118
```